.gitignore

```diff
@@ -16,6 +16,7 @@ cmd/tapd/tapd
 /itest/chantools
 /itest/regtest/*
 /itest/*.log
+/itest/*.log.*
```

Comment:
nit: was the above line just ineffective? in that case could only keep the new pattern

Reply:
I think that in `.gitignore` files, patterns are interpreted using globbing syntax, not regular expressions.

And therefore `/itest/*.log.*` does not match on `/itest/file.log`